nidaqmx/libnidaqmx.py

```diff
@@ -780,6 +818,45 @@ def __init__(self, name = ""):
         self.sample_mode = None
         self.samples_per_channel = None
 
+        Task.load_maps()
+
+    @staticmethod
+    def load_maps():
+        """
+        Initializes some class variables that are used in several functions.
+
+        Initializing these class variables via a function allows for the
+        nidaqmx_h file to be loaded later.  This eases the implementation of a
+        simulated low-level library.
+        """
+        if 'sample_mode_map' in dir(Task):
+          return
+        Task.sample_mode_map = dict (finite = DAQmx.Val_FiniteSamps,
```

Comment:
Could loading the mappings be implemented without introducing the static method load_maps? (to me load_maps approach looks hackish).

For instance, the attributes sample_mode_map, edge_map, sample_timing_type_map as well as their computed reverses, could all be simply class attributes.
The late loading of nidaqmx_h does not seem to be implemented anyway. If that is the plan, then the mappings could be introduced as instance properties.